package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/court-order-poa": "1.0.11",
     "@bcrs-shared-components/enums": "1.0.8",
     "@bcrs-shared-components/interfaces": "1.0.17",
-    "@bcrs-shared-components/staff-comments": "1.0.3",
+    "@bcrs-shared-components/staff-comments": "1.1.0",
```

Comment:
FYI, I went from 1.0.x to 1.1.0 due to a change that may be incompatible in some apps. (Ie, the app has to provide the updated date mixin, so the minor version change should be a trigger to verify this when importing the updated component.)